src/components/MenuBar/MenuBar.tsx

```diff
@@ -11,16 +11,18 @@ import Toolbar from '@material-ui/core/Toolbar';
 import FormControl from '@material-ui/core/FormControl';
 import InputLabel from '@material-ui/core/InputLabel';
 import { Offline, Online } from 'react-detect-offline';
+import { TRACK_TYPE, PARTICIANT_TYPE, EROOR_MESSAGE } from '../../utils/displayStrings';
 
 import LocalAudioLevelIndicator from './LocalAudioLevelIndicator/LocalAudioLevelIndicator';
 import ToggleFullscreenButton from './ToggleFullScreenButton/ToggleFullScreenButton';
 import ToggleGridViewButton from './ToggleGridViewButton/ToggleGridViewButton';
 import SettingsButton from './SettingsButton/SettingsButton';
-
 import { useAppState } from '../../state';
 import useRoomState from '../../hooks/useRoomState/useRoomState';
 import useVideoContext from '../../hooks/useVideoContext/useVideoContext';
 
+const JOIN_ROOM = 'Join Room';
```

Comment:
If this is a message, maybe we should add suffix such as _MESSAGE or _LABEL.
Can we make all messages in associative array? i.e. JOIN_ROOM_MESSAGE['en-US'] = 'Join Room';
Can we move all those _MESSAGES / _LABELS to separate resource file (i.e. displayStrings.ts)?

Reply:
?